internal/config/worker_config.go

```diff
@@ -23,10 +19,6 @@ func LoadWorkerConfig() Config {
 	}
 
 	return Config{
-		DBUser: os.Getenv("DATABASE_USER"),
-		DBPassword: os.Getenv("DATABASE_PASSWORD"),
-		DBName: os.Getenv("DATABASE_NAME"),
-		DBSslMode: os.Getenv("DATABASE_SSL_MODE"),
 		RQUrl: os.Getenv("RABBITMQ_URL"),
```

Comment:
Lets do it another way, lets assemble the link directly in the config. This will secure us from potential bugs later. So in env you should have RABBITMQ_HOST, PORT, USER, PASSWORD. Retrive them, and store as one property in config struct. There is no need to store them seperatly, only the assembled url.